### PR TITLE
Fix sort related bugs

### LINKED ISSUE #200: After sort freq, added issue was inserted to the middle of issue list

### LINKED ISSUE #207: Edit reset Search Frequency
After edit the searchFrequency become 0 because of incorrect constructor called.

### PR DESCRIPTION
**Pull request by YShu7**
Fix #200 , #202 
Fix #207 
Add `createdTime` to `Issue` to make sure the list follows default order if Comparator returns 0

### CODE REVIEW

File: src/main/java/seedu/saveit/model/issue/IssueChroComparator.java [hunk 1]
```diff
@@ -8,7 +8,26 @@
...
  */
 public class IssueChroComparator implements Comparator<Issue> {
+    /**
+     * Compare Issue a and b with their lastModifiedTime.
+     * If lastModifiedTime are the same, sort by created time.
+     */
     public int compare(Issue a, Issue b) {
-        return b.getLastModifiedTime().compareTo(a.getLastModifiedTime());
+        int timeDiff = b.getLastModifiedTime().compareTo(a.getLastModifiedTime());
+        if (timeDiff == 0) {
```
Comment (leo-1997) [inline src/main/java/seedu/saveit/model/issue/IssueChroComparator.java]: it is good to do bounds checking and provides a default sort order

File: src/main/java/seedu/saveit/model/issue/IssueFreqComparator.java [hunk 2]
```diff
@@ -8,7 +8,26 @@
  * Comparator used to sort the Issues in search frequency order
  */
 public class IssueFreqComparator implements Comparator<Issue> {
+    /**
+     * Compare Issue a and b with their searchFrequency.
+     * If searchFrequency are the same, sort by created time.
+     */
     public int compare(Issue a, Issue b) {
-        return b.getFrequency().compare(a.getFrequency());
+        int freqDiff = b.getFrequency().compare(a.getFrequency());
```
Comment (leo-1997) [inline src/main/java/seedu/saveit/model/issue/IssueFreqComparator.java]: Same as here. I think UG and DG also need to be updated to state that the created time will be the back up order

File: src/main/resources/view/ResultDisplay.fxml [hunk 3]
```diff
@@ -5,6 +5,6 @@
 <?import javafx.scene.layout.StackPane?>
 <StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1">
   <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" />
-    <Label fx:id="directory" styleClass="result-display" StackPane.alignment="BOTTOM_RIGHT">
-    </Label>
+  <Label fx:id="sortType" styleClass="status-msg" StackPane.alignment="BOTTOM_LEFT"/>
```
Comment (leo-1997) [inline src/main/resources/view/ResultDisplay.fxml]: occasionally, the sort type will be overlapped with other result display message. Might need to be improved

Comment (YShu7) [reply src/main/resources/view/ResultDisplay.fxml]: Will fix in next PR

**Pull request merged**